src/bosh-google-cpi/action/create_vm.go

```diff
@@ -113,6 +118,17 @@ func (cv CreateVM) Run(agentID string, stemcellCID StemcellCID, cloudProps VMClo
 		return "", bosherr.WrapError(err, "Creating VM")
 	}
 
+	var bs BackendService
+	if bsString, ok := cloudProps.BackendService.(string); ok {
+		bs = BackendService{bsString, ""}
```

Comment:
Let's name the fields when initializing the struct to keep it consistent and easier to read (i.e., BackendService{name: ..., scheme: ...,}